packages/tiktok-video-element/tiktok-video-element.js

```diff
@@ -0,0 +1,252 @@
+class TikTokVideoElement extends (globalThis.HTMLElement ?? class {}) {
+  static EMBED_BASE = 'https://www.tiktok.com/player/v1';
+  static observedAttributes = [
+    'video-id',
+    'controls',
+    'progress_bar',
+    'play_button',
+    'volume_control',
+    'fullscreen_button',
+    'timestamp',
```

Comment:
we won't support non native htmlvideoelement attributes. except for `config` which is a property only, see youtube-video and vimeo-video